src/Symfony/Component/DependencyInjection/Tests/Fixtures/CheckTypeDeclarationsPass/ExpectsIntegerArgument.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symfony\Component\DependencyInjection\Tests\Fixtures\CheckTypeDeclarationsPass;
+
+final readonly class ExpectsIntegerArgument
+{
+    public function __construct(public int $foo) {}
+
+    public static function create(int $foo): self
+    {
+        return new self($foo);
+    }
+
+    public function instance(): self {
```

Comment:
```suggestion
    public function instance(): self
    {
```

Reply:
This should be caught by CI (Fabbot), please don't waste your precious time for being Human Linter 🙈.

(thanks for the review anyway 🍻)

Reply:
fabbot ignores the `Fixtures` folder because we have many fixtures that are actually about generated code (used for comparisons for expectations) or about cases for which we want to explicitly test things that don't match the core coding styles for some reasons.